dev/com.ibm.ws.jaxrs.defaultexceptionmapper/src/com/ibm/ws/jaxrs/defaultexceptionmapper/package-info.java

```diff
@@ -0,0 +1,15 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+/**
+ * @version 1.0
+ */
+@org.osgi.annotation.versioning.Version("1.0")
```

Comment:
This should include `@TraceOptions` - take a look at some of the other JAX-RS package-info.java files for examples.